airflow/cli/commands/role_command.py

```diff
@@ -41,3 +39,259 @@ def roles_create(args):
     appbuilder = cached_app().appbuilder  # pylint: disable=no-member
     for role_name in args.role:
         appbuilder.sm.add_role(role_name)
+
+
+mapping = {
+    # ========================= Airflow Views
+    ("Airflow", "can_blocked"): [("Dag", "can_read"), ("DagRun", "can_read")],
+    ("Airflow", "can_clear"): [("Dag", "can_read"), ("Task", "can_read"), ("TaskInstance", "can_delete")],
+    ("Airflow", "can_code"): [("Dag", "can_read"), ("DagCode", "can_read")],
+    ("Airflow", "can_dag_details"): [("Dag", "can_read"), ("DagRun", "can_read")],
+    ("Airflow", "can_dag_stats"): [("Dag", "can_read"), ("DagRun", "can_read")],
+    ("Airflow", "can_dagrun_clear"): [
+        ("Dag", "can_read"),
+        ("Task", "can_read"),
+        ("TaskInstance", "can_delete"),
+    ],
+    ("Airflow", "can_dagrun_failed"): [("Dag", "can_read"), ("DagRun", "can_edit")],
+    ("Airflow", "can_dagrun_success"): [("Dag", "can_read"), ("DagRun", "can_edit")],
+    ("Airflow", "can_delete"): [("Dag", "can_delete")],
+    ("Airflow", "can_duration"): [("Dag", "can_read"), ("TaskInstance", "can_read"), ("Task", "can_read")],
+    ("Airflow", "can_extra_links"): [("Dag", "can_read"), ("Task", "can_read")],
+    ("Airflow", "can_failed"): [("Dag", "can_read"), ("Task", "can_read"), ("TaskInstance", "can_edit")],
+    ("Airflow", "can_gantt"): [("Dag", "can_read"), ("Task", "can_read"), ("TaskInstance", "can_read")],
+    ("Airflow", "can_get_logs_with_metadata"): [
+        ("Dag", "can_read"),
+        ("TaskInstance", "can_read"),
+        ("Log", "can_read"),
+    ],
+    ("Airflow", "can_graph"): [
+        ("Dag", "can_read"),
+        ("TaskInstance", "can_read"),
+        ("Task", "can_read"),
+        ("Log", "can_read"),
+    ],
+    ("Airflow", "can_index"): [("Dag", "can_read")],
+    ("Airflow", "can_landing_times"): [
+        ("Dag", "can_read"),
+        ("TaskInstance", "can_read"),
+        ("Task", "can_read"),
+    ],
+    ("Airflow", "can_last_dagruns"): [("Dag", "can_read"), ("DagRun", "can_read")],
+    ("Airflow", "can_log"): [("Dag", "can_read"), ("TaskInstance", "can_read"), ("Log", "can_read")],
+    ("Airflow", "can_paused"): [("Dag", "can_paused")],
+    ("Airflow", "can_redirect_to_external_log"): [
+        ("Dag", "can_read"),
+        ("TaskInstance", "can_read"),
+        ("Log", "can_read"),
+    ],
+    ("Airflow", "can_refresh"): [("Dag", "can_edit")],
+    ("Airflow", "can_refresh_all"): [("Dag", "can_edit")],
```

Comment:
These are actually just read-operations (it refreshes from the DB ensuring there's no stale cache). It may go away soon anyway, but it isn't really an "edit"